core/src/main/java/org/mobilitydata/gtfsvalidator/notice/LeadingOrTrailingWhitespacesNotice.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Note that this has a dependency on https://github.com/MobilityData/gtfs-validator/pull/729 as both commits and included here (although I believe the new test for NonAsciiOrNonPrintableCharNotice is missing). Can that dependency be removed so we can merge this separately?

Reply:
Done. My bad, GitHub is a bit challenging when you have dependent pull requests :)